Makefile

```diff
@@ -69,32 +67,63 @@ vet: ## Run go vet against code.
 
 .PHONY: test
 test: manifests generate fmt vet envtest ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test ./... -coverprofile cover.out
+
+.PHONY: e2e
+e2e: ## Runs e2e tests
+	e2e/e2e.sh
+
+.PHONY: kind
+kind: ## Bootstrap Kind Locally
+	sh e2e/kind.sh
+
+## Make Docker build for kind registery
+.PHONY: docker-build-local
+docker-build-local: ## Build docker image with the manager.
+	docker build -t ${IMG_KIND}:${IMG_TAG} .
+
+## Make Docker push locally to kind registery
+.PHONY: docker-push-local
+docker-push-local: ## Build docker image with the manager.
+	docker push ${IMG_KIND}:${IMG_TAG}
 
 ##@ Build
 
 .PHONY: build
-build: generate fmt vet ## Build manager binary.
+build: manifests generate fmt vet ## Build manager binary.
 	go build -o bin/manager main.go
 
 .PHONY: run
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
-# helm lint
-lint: 
-	helm lint ./chart
-
-template:
-	helm -n druid-operator template cluster-druid-operator ./chart --debug
-
+# If you wish built the manager image targeting other platforms you can use the --platform flag.
+# (i.e. docker build --platform linux/arm64 ). However, you must enable docker buildKit for it.
+# More info: https://docs.docker.com/develop/develop-images/build_enhancements/
 .PHONY: docker-build
 docker-build: test ## Build docker image with the manager.
-	docker build -t ${IMG}:${IMG_TAG} .
+	docker build -t ${IMG} .
```

Comment:
why no tag ?

Reply:
Fixed both `make manifests` & docker-build (and docker-push and docker-buildx)